### PR TITLE
Issue 125 Merge private SitePreparationCost into public SitePreparationCost module

### PR DESCRIPTION
**Pull request by parangat94**
This extends siteprep module's capabilities to model distributed wind BOS costs. I ran validation for this module. Please see results below:

siteprep

### CODE REVIEW

File: landbosse/model/SitePreparationCost.py [hunk 1]
```diff
@@ -565,11 +727,19 @@ def outputs_for_detailed_tab(self, input_dict, output_dict):
...
         result.append({
-        'unit': 'm',
-        'type': 'variable',
-        'variable_df_key_col_name': 'Road length',
-        'value': float(self.output_dict['road_length_m'])
+            'unit': 'm',
+            'type': 'variable',
+            'variable_df_key_col_name': 'Road width',
+            'value': self.output_dict['road_width_m']
         })
```
Comment (eberlea) [inline landbosse/model/SitePreparationCost.py:743]: Why was this changed? Does distributed wind not report any road length? Also, it looks like road_width might be duplicated now... Please check.

File: landbosse/tests/model/test_SitePreparationCost.py [hunk 2]
```diff
@@ -13,28 +13,30 @@ class TestSitePreparationCost(TestCase):
     def setUp(self):
```
Comment (eberlea) [inline landbosse/tests/model/test_SitePreparationCost.py:13]: Shouldn't the test case data be set for utility? Maybe we're just planning to go back and revise all of these later. If so, please add an issue on GitHub so we don't forget to do that.

Comment (parangat94) [reply landbosse/tests/model/test_SitePreparationCost.py:13]: @eberlea I think setting this as an issue would work best. Besides, next step after the PRs are approved and merged will focus heavily on testing (based on my team email earlier today). And the testing will cover both utility and DW scale.

File: landbosse/model/SitePreparationCost.py (hunk 1 above)
Comment (parangat94) [reply landbosse/model/SitePreparationCost.py]: @eberlea given the highly varying nature of road lengths in DW projects (including projects with no roads at all), the team couldn't come up with a valid relationship for road length. It made most sense instead, to add road length as a user input (for DW BOS modeling). So basically, in the project_list spreadsheet, the column 'Road Adder (m)' is used as the user defined input for total road length in DW projects. 

Good catch on the double reporting of ['road_width_m']. This has now been fixed.

**Comment by parangat94** [comment]
@akey7 merged `develop` into branch `issue_108_integrate_DW`, followed by merge of `issue_108_integrate_DW` into current branch (`issue_125_integrate_DW_siteprep`). 

PR ready for final review.
